From f26bd2f898b45cc24854031e3a13397f315c2099 Mon Sep 17 00:00:00 2001
From: Sergey Alexandrov <alexandrov88@gmail.com>
Date: Thu, 31 Aug 2017 11:59:19 +0200
Subject: [PATCH] Generate pkgconfig for 2d module

---
 2d/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/2d/CMakeLists.txt b/2d/CMakeLists.txt
index afaa99846..5a3db1040 100644
--- a/2d/CMakeLists.txt
+++ b/2d/CMakeLists.txt
@@ -35,6 +35,9 @@ if(build)
 
     include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include" ${VTK_INCLUDE_DIRECTORIES})
 
+    set(LIB_NAME "pcl_${SUBSYS_NAME}")
+    PCL_MAKE_PKGCONFIG_HEADER_ONLY("${LIB_NAME}" "${SUBSYS_NAME}" "${SUBSYS_DESC}" "${SUBSYS_DEPS}" "" "" "" "")
+
     #Install include files
     PCL_ADD_INCLUDES("${SUBSYS_NAME}" "${SUBSYS_NAME}" ${incs})
     PCL_ADD_INCLUDES("${SUBSYS_NAME}" "${SUBSYS_NAME}/impl" ${impl_incs})
-- 
2.19.0

