From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Fri, 18 Aug 2017 11:30:38 +0200
Subject: Fix PCL_ROOT path

---
 PCLConfig.cmake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/PCLConfig.cmake.in b/PCLConfig.cmake.in
index 9f78fcf..91e6ebd 100644
--- a/PCLConfig.cmake.in
+++ b/PCLConfig.cmake.in
@@ -398,7 +398,7 @@ if(WIN32 AND NOT MINGW)
   get_filename_component(PCL_ROOT "${PCL_DIR}" PATH)
 else()
 # PCLConfig.cmake is installed to PCL_ROOT/share/pcl-x.y
-  get_filename_component(PCL_ROOT "${CMAKE_CURRENT_LIST_DIR}/../.." ABSOLUTE)
+  get_filename_component(PCL_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../../.." ABSOLUTE)
 endif()
 
 # check whether PCLConfig.cmake is found into a PCL installation or in a build tree
